app/src/main/java/org/simple/clinic/drugs/selection/DrugListItem.kt

```diff
@@ -41,12 +44,26 @@ data class ProtocolDrugListItem(
     viewBinding.protocoldrugItemDosage.visibleOrGone(prescribedDrug != null)
     viewBinding.protocoldrugItemDosage.text = prescribedDrug?.dosage
 
+    viewBinding.prescribeddrugItemProtocoldrugRootlayout.shapeAppearanceModel = topMostCardShapeAppearanceModel(holder)
+
     viewBinding.root.setOnClickListener {
       subject.onNext(DrugListItemClicked.PrescribedDrugClicked(drugName, prescribedDrug))
     }
   }
 }
 
+private fun topMostCardShapeAppearanceModel(holder: BindingViewHolder) = if (holder.adapterPosition == 0) {
```

Comment:
Instead of passing in the entire holder, this parameter should take in a boolean named `isTopMostCard` since that is the piece of information that is actually relevant to this method.